src/current/_includes/v23.1/sidebar-data/reference.json

```diff
@@ -1340,14 +123,60 @@
             "urls": [
               "/cockroachcloud/pci-dss.html"
             ]
+          },
+          {
+            "title": "System Catalogs",
+            "items": [
+              {
+                "title": "Overview",
+                "urls": [
+                  "/${VERSION}/system-catalogs.html"
+                    ]
+              },
+              {
+                "title": "<code>crdb_internal</code>",
+                "urls": [
+                  "/${VERSION}/crdb-internal.html"
+                ]
+              },
+              {
+                "title": "<code>information_schema</code>",
+                "urls": [
+                  "/${VERSION}/information-schema.html"
+                ]
+              },
+              {
+                "title": "<code>pg_catalog</code>",
+                "urls": [
+                  "/${VERSION}/pg-catalog.html"
+                ]
+              },
+              {
+                "title": "<code>pg_extension</code>",
+                "urls": [
+                  "/${VERSION}/pg-extension.html"
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Spatial Features",
+        "items": [
+          {
+            "title": "Work with Spatial Data",
+            "urls": [
+              "/${VERSION}/spatial-data.html"
+            ]
           }
         ]
       },
       {
-        "title": "CLI",
+        "title": "Cockroach Commands",
```

Comment:
Based on preview, the contents of the "
`cockroach` commands" submenu still need to be pulled one level higher:

<img width="214" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/441de3c6-1da2-4b2a-9663-65769d0fa5fd">

Here's the IA proposal:

<img width="343" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/46dcedd4-f5ff-4897-b5d5-5a50ea039048">

Also, I'm still uncertain whether this should in fact be `cockroach` Commands and not Cockroach Commands. Did we land either way on style?

Reply:
Updated. 

re: `cockroach` vs `Cockroach`, there's no consensus. I'll file a style issue: https://cockroachlabs.atlassian.net/browse/DOC-8288

For now, I'll use `Cockroach Commands` since it's the least "styled"

Reply:
@taroface - should I move "The `cockroach-sql` command` to be just under `cockroach sql` and standardize the heading to be `cockroach-sql`?

Reply:
My understanding from @ianjevans is that `cockroach-sql` is a standalone binary that has the same functionality as `cockroach sql`. I'm not sure if it technically should be filed as a `cockroach` command since it is `cockroach-sql`. I'd suggest leaving it as is for now.